SixTrack/buildLibraries.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# This script can be used to automatically build the "one time build"
+# libraries boinc and libarchive.
+
+set -e #Exit on error
+
+#Make sure we have the right submodule versions
+git submodule init
+git submodule update
+
+### BOINC
+
+### libArchive
+rm -rf libarchive_build
+mkdir libarchive_build
+cd libarchive_build
+
+cmake -DENABLE_BZip2=ON -DENABLE_CAT=OFF -DENABLE_CPIO=OFF -DENABLE_EXPAT=OFF -DENABLE_INSTALL=OFF -DENABLE_LIBXML2=OFF -DENABLE_LZMA=OFF -DENABLE_NETTLE=OFF -DENABLE_OPENSSL=OFF -DENABLE_TAR=OFF -DENABLE_CNG=OFF -G "Unix Makefiles" ../libarchive -LH
```

Comment:
Hmm, that may be a good idea. Only thing is that this isn't really running within cmake, it's just a shell script to make things reproducible, kindof like cmake_six (which it may eventually be integrated into).

Another possibility would be to somehow put it into the main cmake file, similarly to what was done with crlibm?